src/client/src/pages/settings/admin/userSettings.tsx

```diff
@@ -0,0 +1,223 @@
+import { ChangeEvent, useCallback, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Checkbox, Chip, Stack, Tooltip } from "@mui/material";
+import {
+  DataGrid,
+  GridColDef,
+  GridColumnVisibilityModel,
+  GridFilterModel,
+  GridRenderCellParams,
+  GridToolbar,
+} from "@mui/x-data-grid";
+import { Trash2 } from "lucide-react";
+import { User, WorkgroupRole } from "../../../api/apiInterfaces.ts";
+import { fetchUsers } from "../../../api/user.ts";
+import { theme } from "../../../AppTheme.ts";
+import { muiLocales } from "../../../mui.locales.ts";
+import { TablePaginationActions } from "../../overview/boreholeTable/TablePaginationActions.tsx";
+
+export const UserSettings = () => {
+  const { t, i18n } = useTranslation();
+  const [users, setUsers] = useState<User[]>([]);
+  const [filterModel, setFilterModel] = useState<GridFilterModel>({
+    items: [],
+    quickFilterExcludeHiddenColumns: true,
+    quickFilterValues: [""],
+  });
+  const [columnVisibilityModel, setColumnVisibilityModel] = useState<GridColumnVisibilityModel>({
+    company: false,
+  });
+  const handleFilterModelChange = useCallback((newModel: GridFilterModel) => setFilterModel(newModel), []);
+  const handleColumnVisibilityChange = useCallback(
+    (newModel: GridColumnVisibilityModel) => setColumnVisibilityModel(newModel),
+    [],
+  );
+
+  useEffect(() => {
+    const getUsers = async () => {
+      return await fetchUsers();
+    };
+    getUsers().then(users => setUsers(users));
```

Comment:
PP: Da man bereits eine lokale Funktion verwendet, kann man sich das Mischen von `await` und `then` sparen.
Beispiel:
```
useEffect(() => {
  async function getUsers() {
      const users = await fetchUsers();
      setUsers(users);
  }

  getUsers();
}, []);
```

So könnte man in `getUsers()` auch ein Error-Handling implementieren, wenn man es möchte.

Reply:
Habs gemacht um die IDE Warnung "Promise returned from getUsers is ignored" zu umgehen :D Aber macht natürlich wenig Sinn .
Im nächsten PR bin ich eh grad dran, da Errorhandling zu integrieren für das Get und das Update.